spring-cloud-task-batch/src/test/java/org/springframework/cloud/task/batch/handler/TaskJobLauncherCommandLineRunnerTests.java

```diff
@@ -112,6 +112,17 @@ public void testTaskJobLauncherCLRSuccessFailWithTaskExecutor() {
 				enabledArgs);
 	}
 
+	@Test
+	public void testNoTaskJobLauncher() {
+		String[] enabledArgs = new String[] {
+				"--spring.cloud.task.batch.failOnJobFailure=true",
+				"--spring.cloud.task.batch.failOnJobFailurePollInterval=500",
+				"--spring.batch.job.enabled=false" };
+		this.applicationContext = SpringApplication.run(new Class[] {
+				TaskJobLauncherCommandLineRunnerTests.JobWithFailureConfiguration.class },
+				enabledArgs);
+	}
```

Comment:
What does this verify?  There are no assertions.

Reply:
The app that is executed has 2 jobs and one of them throws an exception.   So in this case we know the app did not execute because there is no exception   thrown.

Reply:
get jobRepository from context and check the jobexecution count.